routes/web.php

```diff
@@ -72,3 +88,23 @@
 
     Route::get('/secretariat/users', 'SecretariatController@list')->name('secretariat.users');
 });
+
+//test emails with urls
+Route::middleware(['auth', 'log'])->group(function () {
+    Route::get('/test_mails/{mail}/{send?}', function ($mail, $send = false) {
+        //to see preview:   /test_mails/Confirmation
+        //to send:          /test_mails/Confirmation/send            
+        if (config('app.debug')) {
+            $user = Auth::user();
```

Comment:
Usually when tests are run, it will be as a dummy user. Would be better to see a whole test example, but can also be moved to another issue.

Reply:
Basically these are made to test the layout and the message of the emails, not for massive testing.

Reply:
okay